src/devices/casparCG.ts

```diff
@@ -467,11 +480,11 @@ export class CasparCGDevice extends DeviceWithState<State> implements IDevice {
 				// create a channel in state if necessary, or reuse existing channel
 				const channel = caspar.channels[mapping.channel] || { channelNo: mapping.channel, layers: {} }
 				channel.channelNo = Number(mapping.channel) || 1
-				channel.fps = 25
+				channel.fps = this.initOptions ? this.initOptions.fps || 25 : 25
```

Comment:
Is this project too old to use the optional chaining? If not, this (and similar lines below) reads easier as `channel.fps = this.initOptions?.fps || 25`, no?

Reply:
Yes, the version of Typescript used does not have optional chaining yet